extensions/src/main/java/org/pentaho/platform/web/http/api/resources/services/SchedulerService.java

```diff
@@ -85,8 +89,8 @@ public Job createJob( JobScheduleRequest scheduleRequest )
 
     // Used to determine if created by a RunInBackgroundCommand
     boolean runInBackground =
-        scheduleRequest.getSimpleJobTrigger() == null && scheduleRequest.getComplexJobTrigger() == null
-            && scheduleRequest.getCronJobTrigger() == null;
+      scheduleRequest.getSimpleJobTrigger() == null && scheduleRequest.getComplexJobTrigger() == null
+        && scheduleRequest.getCronJobTrigger() == null;
```

Comment:
Not sure if this is what you meab, but it is more work (in my opinion unnecessary work) to exclude these logs for non-WN mode.

I will leave this for now, but we can certainly discuss with the team which lifecycle changes it makes sense to log and whether or not to support this for non-WN mode.